client/src/components/Carousel/Carousel.js

```diff
@@ -0,0 +1,126 @@
+import React, { useState } from "react";
+import styles from "../Carousel/Carousel.module.css";
+import "react-responsive-carousel/lib/styles/carousel.min.css";
+import { Carousel } from "react-responsive-carousel";
+import TaskStatusChart from "../TaskStatusChart/TaskStatusChart";
+import PropTypes from "prop-types";
+
+export const InsightsCarousel = ({ taskData }) => {
+  const startedTasks = {
+    early: [],
+    onTime: [],
+    late: [],
+  };
+
+  const EndedTasks = {
```

Comment:
`EndedTasks` should start with a lowercase (i.e. It should be `endedTasks`) rather than an uppercase (for consistency)

Reply:
Resolved: I made this change.